src/main/java/stepdefs/LoginStepDefs.java

```diff
@@ -0,0 +1,33 @@
+package stepdefs;
+
+import businessobjects.User;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.testng.Assert;
+import pageobject.HomePage;
+import pageobject.LoginPage;
+
+
+public class LoginStepDefs {
+
+    @Given("^Login page is opened$")
```

Comment:
two options
or
Login page is opened and required elements displayed
or split into two steps
Login page is opened
Required login elements displayed

Reply:
done